liquibase-standard/src/main/java/liquibase/resource/ClassLoaderResourceAccessor.java

```diff
@@ -67,6 +67,9 @@ protected void configureAdditionalResourceAccessors(ClassLoader classLoader) {
                 PathHandlerFactory pathHandlerFactory = Scope.getCurrentScope().getSingleton(PathHandlerFactory.class);
 
                 for (URL url : urls) {
+                    if (url.toString().endsWith(".pom")) {
```

Comment:
Can you add a comment explaining like "skipping pom files for <some reason>"?

Reply:
Done